crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -0,0 +1,311 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![deny(clippy::arithmetic_side_effects)]
+#![deny(clippy::cast_possible_truncation)]
+// #![deny(unused_crate_dependencies)]
+// #![deny(missing_docs)]
+// #![deny(warnings)]
+
+#[cfg(feature = "alloc")]
+extern crate alloc;
```

Comment:
I htink we don't need these=)

Reply:
Yeah, no only in the root `lib.rs`. Removed them in https://github.com/FuelLabs/fuel-core/pull/2553/commits/9e7658e6314b2b8d696e2c1edd3c0653a988dbf2